### DISCUSSION TITLE
2.23.0

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#1307] [Releases]
- Complete redesign of main and loyalty card view screens
- Material You design for the settings screen
- Fix crash when using "Take a photo" with disabled camera app

<hr /><em>This discussion was created from the release <a href='https://github.com/CatimaLoyalty/Android/releases/tag/v2.23.0'>2.23.0</a>.</em>

**Comment by JVariance** [comment]
It looks great, but please implement the swipe between groups feature again!

**Reply by TheLastProject** [reply]
It could maybe be re-added if someone writes code to add it properly. The old version had way too much accidental activations and it was a very common complaint. It's really hard to get a swipe feature like that right, sadly.

See https://github.com/CatimaLoyalty/Android/issues/699 and https://github.com/CatimaLoyalty/Android/issues/1089 for more info on bugs with the old implementation.
